### PR TITLE
Add arange_

### PR DESCRIPTION
**Pull request by fzimmermann89**
In-place array range along a dimension. 
i.e. filling in the values with the result of a matching arange

Tiny function, but still useful abstraction to describe what is going on. Will be used for setting the acq idx after a reshaping

### CODE REVIEW

**Comment by fzimmermann89** [comment]
would also be open to a name change,
fill_range_?
fill_arange_?

**Comment by fzimmermann89** [comment]
anybody? 🥺

File: tests/utils/test_arange_.py [hunk 1]
```diff
@@ -0,0 +1,21 @@
+import pytest
```
Comment (ckolbPTB) [inline tests/utils/test_arange_.py:3]:
```suggestion
"""Tests for arange_."""

import pytest
```

**Review by ckolbPTB — COMMENTED** [review]
Probably I would go with `fill_arange_ `

**Pull request closed without merging**